Description: Upstream changes introduced in version 1.0-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 cw1sc7g08 (1.0-2) maverick; urgency=low
 .
   * Second release (Closes: #778)
 .
 The person named in the Author field signed this changelog entry.
Author: User <user@ecs.soton.ac.uk>
Bug-Debian: http://bugs.debian.org/778

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- cw1sc7g08-1.0.orig/usr/share/cw1sc7g08/index.php
+++ cw1sc7g08-1.0/usr/share/cw1sc7g08/index.php
@@ -1,4 +1,3 @@
-#!/usr/bin/php
 <?php
 /*
 	6005_cw1_2012 - the Simple PHP RSS Reader for INFO6005
@@ -92,23 +91,26 @@
 			$rss->load($url);
 		
 			$items = $rss->getItems();
-		
+			echo "<br/><br/>";
 			echo "Items From $url \n";
 			echo "===========";
 			for ($j=0;$j<strlen($url);$j++) {
 				echo "=";
+							
 			}
+			
 			echo "\n\n";
 			
 			if (!$items) {
 				echo "FAILED TO LOAD ANY ITEMS\n\n";
 			} else {
-					
+				
+				echo "<br/>";	
 				for ($k=0;$k<$count;$k++) {
 				
 					if ($items[$k]) {
 	
-						echo " * " . $items[$k]["title"] . "\n";
+						echo " * " . $items[$k]["title"] . "<br/>";
 
 					}			
 		
